tests/scripts/thread-cert/backbone/test_mlr_multicast_routing_across_thread_pans.py

```diff
@@ -146,10 +146,14 @@ def test(self):
         # ping MA1 from Host could generate a reply from R1 and R2
         self.assertTrue(self.nodes[HOST].ping(MA1, backbone=True, ttl=5))
         self.simulator.go(WAIT_REDUNDANCE)
+        self.verify_border_routing_counters(self.nodes[PBBR1], {'inbound_multicast': 1, 'outbound_unicast': 1})
+        self.verify_border_routing_counters(self.nodes[PBBR2], {'inbound_multicast': 1, 'outbound_unicast': 1})
 
         # ping MA2 from R1 could generate a reply from Host and R2
         self.assertTrue(self.nodes[ROUTER1].ping(MA2))
         self.simulator.go(WAIT_REDUNDANCE)
+        self.verify_border_routing_counters(self.nodes[PBBR1], {'inbound_unicast': 2, 'outbound_multicast': 1})
```

Comment:
With current MLD implementation via `IPV6_JOIN_GROUP` for MA registered, I would assume PBBR2 itself will also subscribe to the MA2 when R2 registers MA2, so I am thinking there might be 3 responses to R1 via PBBR1?
https://github.com/openthread/openthread/blob/d805b5dd1c9d69d6aef274d5db700db0311ebcc7/src/posix/platform/multicast_routing.cpp#L164

Reply:
is it the reason to use `assertGreaterEqual` other than `assertEqual` in below Line 252?